packages/server/modules/workspaces/events/eventListener.ts

```diff
@@ -37,16 +52,59 @@ export const onProjectCreatedFactory =
     )
   }
 
+export const onInviteFinalizedFactory =
```

Comment:
I think this is controversial/undiscussed behavior? Adding a user to a workspace when they're invited to a workspace project?

It makes sense to me but we definitely should get explicit somewhere that this is how we want things to happen. (I expected the flow to be "invite to workspace" => "have access to workspace projects")

Reply:
@cdriesler This is based on this figjam that Gergo did for me: https://www.figma.com/board/kJgZgku0oW6CfeKMe3HQfI/Workspace-invites?node-id=0-1&t=nUTTufUxovJVwbsp-0

Our reqs are strewn all over the place and not always formalized which is why we're not all on the same page about them haha

Reply:
Totally fair, will try to collect notes today. Ready to approve after the generics change or something like this from you:

![Creating_Bugs_Bunny's__No_](https://github.com/user-attachments/assets/a60bfdf1-73d0-4247-be6b-782434b81083)

Reply:
@cdriesler ye, im workin through all of ur other comments rn 👍